app/controllers/api/motions_controller.rb

```diff
@@ -24,6 +24,12 @@ def update_outcome
     respond_with_resource
   end
 
+  def closed
+    load_and_authorize :group
+    @motions = @group.closed_motions.order(closed_at: :desc)
```

Comment:
Yes!

The idea is that `visible_records` handles the privacy, ie, it's the subset of records that the user has access to, and the block in `instantiate_collection` defines 'what records I want right now', ie, all the motions that are closed.